x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/string/Locate.java

```diff
@@ -46,11 +46,10 @@ public class Locate extends EsqlScalarFunction implements OptionalArgument {
     private final Expression substr;
     private final Expression start;
 
-    @FunctionInfo(
-        returnType = "integer",
-        description = "Returns an integer that indicates the position of a keyword substring within another string.",
-        examples = @Example(file = "string", tag = "locate")
-    )
+    @FunctionInfo(returnType = "integer", description = """
+        Returns an integer that indicates the position of a keyword substring within another string.
```

Comment:
Returns an integer (starting from `1`) that indicates the position of a keyword substring within another string.
Returns `0` if the substring is not found.

Reply:
Many developers are not used with string positions starting from 1. This is why I kind of like to have that as a separate note. If you feel very strongly about I can change this again, but for now I will merge the PR as it is.